pex/bin/pex.py

```diff
@@ -458,10 +458,26 @@ def configure_clp_pex_environment(parser):
         default=[],
         type="str",
         action="append",
-        help="The Python interpreter to use to build the pex.  Either specify an explicit "
-        "path to an interpreter, or specify a binary accessible on $PATH. This option "
-        "can be passed multiple times to create a multi-interpreter compatible pex. "
-        "Default: Use current interpreter.",
+        help=(
+            "The Python interpreter to use to build the PEX (default: current interpreter). This "
+            "cannot be used with `--interpreter-constraint`, which will instead cause PEX to "
+            "search for valid interpreters. Either specify an absolute path to an interpreter, or "
+            "specify a binary accessible on $PATH like `python3.7`. This option can be passed "
+            "multiple times to create a multi-interpreter compatible PEX."
+        ),
+    )
+    group.add_option(
+        "--python-path",
+        dest="python_path",
+        default=None,
+        type="str",
+        help=(
+            "The paths used by PEX to find valid interpreters when `--interpreter-constraint` "
+            "and/or `--resolve-local-platforms` are used (default: the $PATH). Specify a "
+            "colon-separated string containing paths of blessed Python interpreters. Can be "
```

Comment:
I don't think "blessed" is a useful word here (I know it's in the docstring for `PEX_PYTHON_PATH`). What does it mean? And the wording here is rather strained and redundant in general. 

How about something like: 

```
help=(
    "Colon-separated paths to search for interpreters when `--interpreter-constraint` or `--resolve-local-platforms` are specified (default: $PATH). Each element can be the absolute path of an interpreter binary, or a directory containing interpreter binaries.
)
```